test/e2e/mount_test.go

```diff
@@ -641,3 +650,88 @@ func TestMountRotateSecret(t *testing.T) {
 
 	// TODO: Add checks for regional secret
 }
+
+// Execute a test job that mounts a extract secret and checks that the value is correct.
+func TestMountExtractSecret(t *testing.T) {
+	secretData := []byte(`{"user":"admin", "password":"password@1234"}`)
+
+	// Create test secret
+	secretFile := filepath.Join(f.tempDir, "secretExtractValue")
+	check(os.WriteFile(secretFile, secretData, 0644))
+	check(execCmd(exec.Command(
+		"gcloud", "secrets", "create", f.testExtractSecretID,
```

Comment:
This secrets ID is already being used, please use different ID.

Reply:
We've already changed the ID from `testSecretID` to `testExtractSecretID`